src/mapReducers.js

```diff
@@ -26,6 +26,40 @@ function resolveKey(action, keySelector) {
   return resolvedKey || TARGET_ALL_REDUCERS;
 }
 
+function calculateNewTargetAllState(state, action, resolveReducer) {
```

Comment:
Maybe `calculateNewStateForAllKeys`, and I would use the `calculateNewStateForKey` to perform this operation for each key. You could then do the `shallowEquals` on the resulting state to determine if it has changed (`_.eq` should be good enough for that).